src/main/java/dev/openfeature/sdk/ProviderEventDetails.java

```diff
@@ -0,0 +1,18 @@
+package dev.openfeature.sdk;
+
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+import lombok.Data;
+import lombok.experimental.SuperBuilder;
+
+/**
+ * Interface for attaching event handlers.
+ */
+@Data @SuperBuilder(toBuilder = true)
+public class ProviderEventDetails {
+    @Nullable private List<String> flagsChanged;
```

Comment:
Should this be an array or a list?

Reply:
I would prefer having this handle by lombok and stay on the `List`

Reply:
> I would prefer having this handle by lombok and stay on the List

Just to clarify... this is already handled by lombok builder. Is there some other lombok tooling you want added? Or are you just saying  you want it kept as a list.

Reply:
Sorry, I didn't explain my reasoning well. I suggest staying on List because of Lombok. AFAIK, it wouldn't work with arrays :)